bagit.xml

```diff
@@ -283,7 +283,7 @@ document are to be interpreted as described in <xref target="RFC2119"/>.
     "sha1") as described in <xref target="bag-checksum-algorithms"/>.
   </t>
             <t hangText="payload">
-    The data encapsulated by the bag. The contents of the payload
+    The data encapsulated by the bag as a set of named files. The contents of the payload files
```

Comment:
I would also add reference to directories as well. I know that directories are just part of the names of files, but it would make it clearer to readers.

Reply:
Agreed. However directories are not first-class citizens (as they don't have checskums), so I went for the wording "files organized in sub-directories".